### PR TITLE
Use local picture default

### PR DESCRIPTION
**Pull request by hl219**
Use local picture default in 
Profile setting,
view link list,
main activity

### CODE REVIEW

File: app/src/main/java/com/example/rum8/dataModels/LinkListSingleLink.java [hunk 1]
```diff
@@ -37,4 +37,8 @@ public String getUid() {
         return this.uid;
     }
 
+    public void setBitMap(Bitmap bitMap){
```
Comment (cksriprajittichai) [inline app/src/main/java/com/example/rum8/dataModels/LinkListSingleLink.java]: Do you think we should name this something less technical? Like `setPicture` or `setImage`?

File: app/src/main/java/com/example/rum8/fragments/PotentialRoommateProfileDefault.java [hunk 2]
```diff
@@ -77,6 +78,7 @@ public void setUserProfileImage(Bitmap bitmap) {
 
     @Override
     public void goToProfileSettings() {
+        profilePicture.setImageDrawable(ContextCompat.getDrawable(getActivity(), R.drawable.images));
```
Comment (cksriprajittichai) [inline app/src/main/java/com/example/rum8/fragments/PotentialRoommateProfileDefault.java]: Is this correct? This doesn't seem like code that goes in `goToProfileSettings`

Comment (hl219) [reply app/src/main/java/com/example/rum8/fragments/PotentialRoommateProfileDefault.java]: Yes, it is correct. It shows the picture for main page

Comment (hl219) [reply app/src/main/java/com/example/rum8/fragments/PotentialRoommateProfileDefault.java]: Wait, I'm wrong

**Pull request merged**